cmd/system-probe/app/debug.go

```diff
@@ -25,7 +25,7 @@ func init() {
 
 var (
 	debugCommand = &cobra.Command{
-		Use:   "debug [state]",
+		Use:   "debug [module] [state]",
```

Comment:
How is this related to config?

Reply:
It was actually broken before and I discovered it during testing of the config. It wasn't allowing access to any of the module-specific debug endpoints.